src/lib/services/playground-service.ts

```diff
@@ -0,0 +1,124 @@
+import FeatureToggleService from './feature-toggle-service';
+import { SdkContextSchema } from 'lib/openapi/spec/sdk-context-schema';
+import { InMemStorageProvider, Unleash as UnleashClient } from 'unleash-client';
+import { IUnleashServices } from 'lib/types/services';
+import { ALL } from '../../lib/types/models/api-token';
+import { PlaygroundFeatureSchema } from 'lib/openapi/spec/playground-feature-schema';
+import { Logger } from '../logger';
+import { IUnleashConfig } from 'lib/types';
+import { FeatureConfigurationClient } from 'lib/types/stores/feature-strategies-store';
+import { Operator } from 'unleash-client/lib/strategy/strategy';
+import { once } from 'events';
+
+enum PayloadType {
+    STRING = 'string',
+}
+
+type NonEmptyList<T> = [T, ...T[]];
+
+export const offlineUnleashClient = async (
```

Comment:
Should this have it's own file or be moved to utils?

Reply:
Absolutely! Done in 9ba93903bfec9354e9d851dfb876a18058a9b62c